lib/wice_grid.rb

```diff
@@ -216,7 +216,7 @@ def process_loading_query #:nodoc:
 
     def process_params  #:nodoc:
       if this_grid_params
-        @status.merge!(this_grid_params)
+        @status.merge!(this_grid_params.to_unsafe_h)
```

Comment:
I haven't found a better option. We need to convert `ActionController::Parameters` to hash, and we have three options:
- call `to_unsafe_h`
- call `permit!.to_h`
- use strong params and permit only necessary values

The last solution sounds the best, but we cannot tell what are the correct params (especially when we are using search boxes)